### PR TITLE
Add inline assets priority

### PR DESCRIPTION
**Pull request by namaless**
Use md5 from content for discard multiple iterance.

### CODE REVIEW

**Comment by rhukster** [comment]
Can you give me more details about what this issue is your solving with this? Perhaps some example on how i can recreate it?

**Comment by namaless** [comment]
No one issue, i read the file and found on inline_css the priority param, so i get idea to add priority feature to inline assets.
I post the pull request but not need to add, for this use "feature" type :)

**Comment by namaless** [comment]
https://github.com/namaless/grav/commit/e013cf70dbc084b4a4112ae48746e1f82b5bc6ed#diff-036bc801ec8840ea4247592b9ded7a66L298

File: system/src/Grav/Common/Assets.php [hunk 1]
```diff
@@ -298,8 +298,12 @@ public function addJs($asset, $priority = 10, $pipeline = true)
     public function addInlineCss($asset, $priority = 10)
     {
 
-        if (is_string($asset) && !in_array($asset, $this->inline_css)) {
-            $this->inline_css[] = $asset;
+        if (is_string($asset) && !array_key_exists(md5($asset), $this->inline_css)) {
+            $this->inline_css[md5($asset)] = [
```
Comment (mahagr) [inline system/src/Grav/Common/Assets.php]: md5() is used twice for the same key

File: system/src/Grav/Common/Assets.php [hunk 2]
```diff
@@ -312,14 +316,19 @@ public function addInlineCss($asset, $priority = 10)
...
      * @return $this
      */
-    public function addInlineJs($asset)
+    public function addInlineJs($asset, $priority = 10)
     {
 
-        if (is_string($asset) && !in_array($asset, $this->inline_js)) {
-            $this->inline_js[] = $asset;
+        if (is_string($asset) && !array_key_exists(md5($asset), $this->inline_js)) {
+            $this->inline_js[md5($asset)] = [
```
Comment (mahagr) [inline system/src/Grav/Common/Assets.php]: md() x2

**Pull request merged**